README.md

```diff
@@ -1,6 +1,6 @@
 # compute-sdk-go
 
-Experimental Go SDK for building [Compute@Edge](https://www.fastly.com/products/edge-compute/serverless) applications with [Go](https://go.dev) (1.21+) and [TinyGo](https://tinygo.org/) (0.28.1+).
```

Comment:
`https://www.fastly.com/products/edge-compute/serverless` is a 301 redirect to `https://www.fastly.com/products/edge-compute`. May be worth updating.

Reply:
https://github.com/fastly/compute-sdk-go/pull/79/commits/36d8fed2dcbd45772aaa8b63c9e196ecf9fb5cc9